cmd/agent/app/reporter/grpc/collector_proxy.go

```diff
@@ -25,6 +25,7 @@ import (
 	"google.golang.org/grpc/resolver/manual"
 
 	"github.com/jaegertracing/jaeger/cmd/agent/app/httpserver"
+	mGrpc "github.com/jaegertracing/jaeger/cmd/agent/app/httpserver/grpc"
```

Comment:
Just curious, what does `m` stands for? Why not `a` (agent)? (no need to change, really just curious)

Reply:
managerGrpc